tests/requirements.txt

```diff
@@ -3,4 +3,6 @@
 cryptography>=41.0.0
 pytest
 pytest-cov
+conda>=23.7
```

Comment:
That's a big jump, any specific reason?

Reply:
It was installing conda 4

Reply:
I tested this yesterday (see #77) and conda 22.11 is sufficient

Reply:
continue here https://github.com/conda/conda-content-trust/pull/75#discussion_r1302039760